app/package.json

```diff
@@ -6,7 +6,7 @@
   "homepage": ".",
   "productName": "Headlamp",
   "scripts": {
-    "compile-electron": "babel electron --out-dir electron/ --extensions '.ts'",
+    "compile-electron": "babel electron --out-dir electron/ --extensions .ts",
```

Comment:
@illume How is this probelmatic, I tried with both ways without and with quotes and it works

Reply:
It runs, but does not compile the scripts (reports 0 compiled and should be something like 4). Note, this is when using cmd.

Reply:
aah I was using the wrong shell, with cmd it is a problem